packages/cli/src/tools/windows/androidWinHelpers.ts

```diff
@@ -0,0 +1,301 @@
+import {readFile, writeFile} from 'fs';
+import {join} from 'path';
+import {executeCommand} from './executeWinCommand';
+import {getProcessorType} from '../processorType';
+import {promisify} from 'util';
+
+const readFileAsync = promisify(readFile);
+const writeFileAsync = promisify(writeFile);
+
+type HypervisorStatus = {
+  hypervisor: 'WHPX' | 'HAXM' | 'AMDH' | 'none';
+  installed: boolean;
+};
+
+/**
+ * Returns the path to where all Android related things should be installed
+ * locally to the user.
+ */
+export const getUserAndroidPath = () => {
+  return join(process.env.LOCALAPPDATA || '', 'Android');
+};
+
+/**
+ * Deals with ANDROID_HOME, ANDROID_SDK_ROOT or generates a new one
+ */
+export const getAndroidSdkRootInstallation = () => {
+  // TODO: Check if the path exists?
```

Comment:
I guess that would make sense, yeah, if this function returning a path is meant to indicate an actual installation exists.